snitun/utils/aiohttp_client.py

```diff
@@ -59,40 +53,26 @@ def wait(self) -> asyncio.Future[None]:
     async def start(
         self,
         whitelist: bool = False,
-        endpoint_connection_error_callback: Callable[[], Coroutine[Any, Any, None]]
```

Comment:
This is a breaking change to remove this but its not needed anymore. Likely it will break mocking in hass_nabucasa but simple enough to remove